app/routes/photos/components/GalleryEditor.js

```diff
@@ -256,19 +257,20 @@ class GalleryEditor extends Component<Props, State> {
             justifyContent="flex-end"
           >
             {!isNew && (
-              <Button
-                onClick={this.onDeleteGallery}
-                className={styles.deleteButton}
+              <ConfirmModalWithParent
+                title="Slett album"
+                message="Dette vil slette hele albumet og alle bildene inni"
```

Comment:
"inni" sounds very casual, how about "alle bildene det inneholder"? Only if you agree, of course. Also, you should probably add a followup à la "...inneholder. Er du sikker?", since the buttons are "Avbryt" and "Ja" which doesn't make that much sense without having a question first.

Reply:
Agreed